teuthology/orchestra/connection.py

```diff
@@ -104,8 +104,7 @@ def connect(user_at_host, host_key=None, keep_alive=False, timeout=60,
                 try:
                     ssh.connect(**connect_args)
                     break
-                except paramiko.AuthenticationException:
-                    log.exception(
-                        "Error connecting to {host}".format(host=host))
+                except paramiko.AuthenticationException as e:
+                    log.error(f"Error authenticating with {host}: {str(e)}")
```

Comment:
Isn't _str_ applied inclusively when the f-str used?

Reply:
Dunno, I didn't want to find out the hard way :)